cmd/fpga_admissionwebhook/README.md

```diff
@@ -138,13 +120,41 @@ $ ./scripts/webhook-deploy.sh --ca-bundle-path /var/run/kubernetes/server-ca.crt
 
 # Mappings
 
-Requested FPGA resources are translated to AF resources. For example,
-`fpga.intel.com/arria10.dcp1.1-nlb0` is translated to `fpga.intel.com/af-d8424dc4a4a3c413f89e433683f9040b`.
+Mappings is a an essential part of the setup that gives a flexible instrument to a cluster
+administrator to manage FPGA bitstreams and to control access to them. Being a set of
+custom resource definitions they are used to configure the way FPGA resource requests get
+translated into actual resources provided by the cluster.
+
+For the following mapping
+
+```yaml
+apiVersion: fpga.intel.com/v1
+kind: AcceleratorFunction
+metadata:
+  name: arria10.dcp1.2-nlb0-preprogrammed
+spec:
+  afuId: d8424dc4a4a3c413f89e433683f9040b
+  interfaceId: 69528db6eb31577a8c3668f9faa081f6
+  mode: af
+```
 
-In orchestrated mode, `fpga.intel.com/arria10.dcp1.1-nlb0` gets translated to
-`fpga.intel.com/region-9926ab6d6c925a68aabca7d84c545738`, and, the corresponding AF IDs are set in
-environment variables for the container. The [FPGA CRI-O hook](../fpga_crihook/README.md)
-then loads the requested bitstream to a region before the container is started.
+requested FPGA resources are translated to AF resources. For example,
+`fpga.intel.com/arria10.dcp1.2-nlb0-preprogrammed` is translated to
+`fpga.intel.com/af-695.d84.aVKNtusxV3qMNmj5-qCB9thCTcSko8QT-J5DNoP5BAs` where the `af-`
+prefix indicates the plugin's mode (`af`), `695` is the first three characters of
+the region interface ID, `d84` is the first three characters of the accelerator function ID
+and the last part `aVKNtusxV3qMNmj5-qCB9thCTcSko8QT-J5DNoP5BAs` is a base64-encoded concatenation
+of the full region interface ID and accelerator function ID.
+The format of resource names (e.g. `arria10.dcp1.2-nlb0-preprogrammed`) can be any and is up
+to a cluster administrator.
+
+The same mapping, but with its mode field set to `region`, would translate
+`fpga.intel.com/arria10.dcp1.2-nlb0-preprogrammed` to `fpga.intel.com/region-69528db6eb31577a8c3668f9faa081f6`,
```

Comment:
copy-paste error?

Reply:
No, that's correct: changing mode doesn't change the mapping's name.